app/src/main/java/pl/pwr/zpi/bcycle/mobile/BCycleNavigationDrawerActivity.kt

```diff
@@ -88,6 +94,13 @@ abstract class BCycleNavigationDrawerActivity: AppCompatActivity() {
             if (it.itemId == R.id.nav_logout) {
                 auth.signOut()
                 finishAndRemoveTask()
+            } else if (it.itemId == R.id.nav_create_group_trip) {
+                startActivity(Intent(applicationContext, TripCreationActivity::class.java))
+                finishAndRemoveTask()
```

Comment:
This might be done as a fragment, although it’s not necessary IMO, because it’s a separate, modal UI. `finishAndRemoveTask` breaks the back button from trip creation to home.